src/components/pages/gfcrPages/GfcrIndicatorSetForm/GfcrIndicatorSetForm.js

```diff
@@ -22,18 +21,6 @@ const StyledForm = styled.form`
   width: 100%;
 `
 
-const handleInputBlur = (formik, event, fieldName, isRoundTo1DP = false) => {
-  const { value } = event.target
-  if (value.trim() === '') {
-    setInputToDefaultValue(formik, fieldName)
-  }
-  const valueAsNumber = Number(value)
-
-  if (value && isRoundTo1DP && !Number.isInteger(valueAsNumber)) {
-    formik.setFieldValue(fieldName, Number(roundToOneDecimal(valueAsNumber)))
-  }
-}
-
```

Comment:
It was unclear what was happening where this was called (onBlur events within child components). We now call more descriptively named functions to restrict decimal places for input values or replace user inputted blanks with previously saved/default values.

 Also the functionality can be called directly from generic libraries and doesnt need to live within a parent.